### DISCUSSION TITLE
Is the octokit.github.io homepage supposed to redirect to docs.github.com?

### DISCUSSION THREAD

**Discussion by chiedo** [octokit/octokit.js#2041] [General]
Hey @gr2m! 

We just noticed that the https://octokit.github.io/ homepage redirects to docs.github.com. Is that by accident or by design?

**Comment by gr2m** [comment]
It was the way before I started working on the JavaScript Octokit in 2017, I don't think anyone thought about it for a long time

**Reply by chiedo** [reply]
Thanks @gr2m!
